controller/src/main/java/org/jboss/as/controller/capability/RuntimeCapability.java

```diff
@@ -106,8 +105,8 @@ private RuntimeCapability(String baseName, Class<?> serviceValueType, ServiceNam
         this.runtimeAPI = runtimeAPI;
         this.serviceValueType = serviceValueType;
         if (serviceValueType != null) {
-            Assert.assertNotNull(baseServiceName);
-            this.serviceName = dynamicElement == null ? baseServiceName : baseServiceName.append(dynamicElement);
+            assert baseServiceName != null;
```

Comment:
What does motivate you to replace the uses of Assert.assertNotNull?

`Assert.assertNotNull` will log a message and it is widely used.

Reply:
@yersan The second assert was a native one. But you're right both should be Assert.assert based then.

Reply:
@yersan I have reverted to Assert.assertNotNull. There is no assertNull equivalent available.